docs/model_control_mode.md

```diff
@@ -0,0 +1,68 @@
+# Model Control Mode
+
+TorchServe now supports model control mode with two settings "none"(default) and "enabled"
+
+## Two ways to set Model Control
+1. Add `--model-api-enabled` to command line when running TorchServe to switch from none to enabled mode. Command line cannot be used to set mode to none, can only be used to set to enabled
+2. Add `model_control_mode=enabled` or `model_control_mode=none` to config.properties file
```

Comment:
To simplify this configuration, please change it to `model_api_enabled` or `model_control_api_enabled` and accept boolean values.